### ISSUE TITLE
Fix MD001: Fix heading level increments for accessibility violations

### ISSUE THREAD

**Issue by adegeo** [dotnet/docs-desktop#2194]
Lots of files in the repo are from the MSDN days when things didn't follow H1 -> H2 -> H3 and instead would jump H1 -> H3 on short articles

**Comment by adegeo** [comment]
Duplicate of #29
